appscale/tools/templates/AppScalefile-cloud

```diff
@@ -42,6 +46,18 @@ zone : 'us-east-1b' # For Amazon EC2
 # for your GCE instances. By default, we randomly generate a group for you.
 #group : 'appscalegroup'
 
+# The Azure resource group name that should be used for all the machines in
+# this AppScale deployment.
+#resource_group: 'appscalegroup'
```

Comment:
It would make more sense to me if these keys were grouped under a main infrastructure key. Although the comment makes it clear that it's Azure-specific, I think the keys could be clearer. Since we don't group subproperties for other infrastructures under the infrastructure key, perhaps prefixing the infrastructure-specific keys could be a short term alternative. For example, this key could be `azure_resource_group`.